src/java.base/share/classes/jdk/internal/access/JavaLangAccess.java

```diff
@@ -378,4 +378,8 @@
      * @see java.lang.invoke.MethodHandles.Lookup#defineHiddenClass(byte[], boolean, MethodHandles.Lookup.ClassOption...)
      */
     Object classData(Class<?> c);
+
+    Module addEnableNativeAccess(Module m);
+
+    boolean isEnableNativeAccess(Module m);
```

Comment:
Probably better to co-locate these with the other module methods, and add a comment so that it's consistent with the existing methods.

Reply:
sure I'll fix this